nightwatch.conf.js

```diff
@@ -137,10 +142,11 @@ module.exports = {
       // More info on configuring capabilities can be found on:
       // https://www.browserstack.com/automate/capabilities?tag=selenium-4
       desiredCapabilities: {
+        build: 'nightwatch-website-test',
         'bstack:options': {
           local: 'false',
-          userName: '${BROWSERSTACK_USER}',
-          accessKey: '${BROWSERSTACK_KEY}',
+          userName: process.env.BROWSERSTACK_USER || '${BROWSERSTACK_USER}',
+          accessKey: process.env.BROWSERSTACK_KEY || '${BROWSERSTACK_KEY}',
```

Comment:
Nightwatch Internally takes env variables into account this won't be necessary.
For reference: [line](https://github.com/nightwatchjs/nightwatch/blob/5901396607ecec5b5d8cf6be59eff3bd1301dcf6/lib/transport/browserstack.js#L43)

Reply:
Make sense, removing it 👍